.circleci/config.yml

```
@@ -288,12 +288,12 @@ jobs:
       - *no_host_check
       - restore_cache:
           keys:
-            - root-npm-v4-{{ checksum "common/config/rush/pnpm-lock.yaml" }}-2
-      - run: sudo npm install -g pnpm @microsoft/rush
+            - root-npm-v4-{{ checksum "common/config/rush/pnpm-lock.yaml" }}-3
+      - run: sudo npm install -g js-yaml@3.13.1 @octokit/rest semver@6.3.0 shelljs@0.8.3 simple-git@1.126.0
       # Identify the committer i.e. CircleCI deployment bot
       - run: git config --global user.email "circleci@example.com"
       - run: git config --global user.name "CircleCI Deployment Bot"
-      - run: node common/scripts/install-run-rushx.js release-tag
+      - run: export NODE_PATH=$(npm root --quiet -g) && node ./scripts/release-tag.js
```

```suggestion
      - run: export NODE_PATH=$(npm root --quiet -g) && node ./scripts/release-tag.js
```
I guess this is the same as `scripts/release-tag.js` just want to make sure it works